spec/spec_helper.rb

```diff
@@ -20,6 +20,8 @@
   require 'simplecov'
   SimpleCov.start 'rails'
   require 'capybara/rspec'
+  require 'webmock'
+  require 'vcr'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.